### DISCUSSION TITLE
获取印象笔记授权失败，不弹出印象笔记登录界面（附解决方案）

### DISCUSSION THREAD

**Discussion by Mickychen00** [Kenshin/simpread#2791] [重要通知]
RT

**Comment by Kenshin** [comment]
嗨，因为印象笔记的一台服务器网络不稳定造成的，你可以 **稍后再试** 。

## 解决方案一

> 请开 **开全局**。


## 解决方案二

> 通过 Chrome SwitchOmega **全局**，同时发现对于保存笔记失败的情况，可以将 `*.simpread.herokuapp.com` 添加进 SwitchOmega 通配符中，这样就不用开全局了 via https://github.com/Kenshin/simpread/issues/3038#issuecomment-964977442

**Reply by P4PER71GER** [reply]
请问是开全局指的是#1567吗？最近试了不下7、8次了，还是没有跳转至授权页面。

**Reply by P4PER71GER** [reply]
懂了，开了全局魔法，已经跳转至授权页面了

**Reply by P4PER71GER** [reply]
但是保存至印象笔记的时候，PAC都不行，还是要开全局。。。这有点麻烦

**Reply by Kenshin** [reply]
> 但是保存至印象笔记的时候，PAC都不行，还是要开全局。。。这有点麻烦

嗨，我个人不建议使用印象笔记功能，因为它并不完美（不是简悦的问题）细节请看 #302，如果你是印象笔记的重度使用者，请使用它自己的剪藏工具。
